dev/test/index.ts

```diff
@@ -262,7 +262,8 @@ const allSupportedTypesInput = {
   bytesValue: Buffer.from([0x1, 0x2]),
 };
 
-const allSupportedTypesOutput = {
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const allSupportedTypesOutput: {[field: string]: any} = {
```

Comment:
This can probably be `unknown`.

Reply:
We need `any` here since we access `pathValue.formattedName` on L680, which would would error since `pathValue` has `unknown` type.

Reply:
You can cast there if it happens just one. `unknown as X` is more type safe than `any`.

Reply:
Good point, done for both.